### ISSUE TITLE
Not seeing the network

### ISSUE THREAD

**Issue by ipodishima** [richonguzman/PiBackup#3]
Hi!

I'm trying to get your thing running, but once I reboot after the installation, I do not see the network I'm supposed to connect to.
Is there a way to enable some kind of troubleshooting / logs?

Thanks!

**Comment by richonguzman** [comment]
Hi, This was working a few years ago, maybe its time to check if this still works fully


did you install the same OS version mentioned or the last ones?

**Comment by ipodishima** [comment]
Oh thank you for answering promptly
Yes, installed `Raspberry Pi OS (Legacy) with desktop` from https://www.raspberrypi.com/software/operating-systems/
Also, I had to force my user to be named `pi` 

One of the errors I have is after rebooting, I get `wlan0 not associated` 
Tried to get some help from ChatGPT to debug (not a daily user of Raspberry) but no luck

**Comment by richonguzman** [comment]
did you get at least a Wifi created to connect from phone or not at all?

**Comment by ipodishima** [comment]
Not at all.
When it reboots, I lose the connection to my local network connected to internet
And I'm not seeing the PiBackup network (kept the default values)

**Comment by richonguzman** [comment]
You should see a new wifi network (and it should be disconnected from your local internet connection as its intended to work without any other connection but his own , to be accessible from the phone)


PS : I saw you code for iOS ??

**Comment by ipodishima** [comment]
Yes, but I'm not seeing it. Is there somewhere in the logs I could check for the wlan PiBackup being setup upon reboot? So I could track if something is wrong?

Yes, I'm a full-time iOS developer :)

**Comment by richonguzman** [comment]
i will have to test it again for new stuff.... let me test it in the next days and I come back to you,

how can I contact you for some tips for iOS dev?

**Comment by ipodishima** [comment]
Okay thanks!

I’ll check it out on tomorrow to see if I can be of any help. 
Also wrote you an email so you have mine!

**Comment by ipodishima** [comment]
Hey there! So I finally made it working. It's because I'm using an old Raspberry which is not supporting the 5GHz so I switched to another channel not being 36
